### PR TITLE
Add SCSS support. Add namespace to CSS vars and `core` components.

### PR DESCRIPTION
**Pull request by kendrick**
Add `cds-` namespace to CSS vars. Enable SCSS usage for the project.

### CODE REVIEW

**Comment by kendrick** [comment]
@janiekyu, could you take a quick look?

* `test-and-lint` is failing because the DOM doesn’t match due to the namespaced class names
* Percy snapshots have very slight (pixel/subpixel) differences that seem to be related to the `Icon` additions

File: packages/core/scripts/build-sass.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+for SCSS in src/components/**/*.scss; do
+  OUTPUT=${SCSS%.scss}.module.css
+  HEADER="\
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */\n\
+/* THIS IS GENERATED CODE. Any changes you make will be overwritten! */\n\
```
Comment (janiekyu) [inline packages/core/scripts/build-sass.sh:7]: Thank you for this =)

File: lerna.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
   "npmClient": "npm",
   "useWorkspaces": true,
-  "version": "1.0.0"
+  "version": "2.0.0-alpha.0"
```
Comment (janiekyu) [inline lerna.json:4]: What does changing the version number here do? I thought it was automatically incremented when we publish to NPM.

File: packages/core/src/components/Form/Checkbox.stories.js [hunk 3]
```diff
@@ -20,25 +22,36 @@ const Checkbox = ({
   const hasError = state === 'error';
   const checkedAttr = checked ? 'checked' : '';
 
+  const storyId = `use-a-more-meaningful-id__d:${
+    disabled.length > 0 ? 'true' : 'false'
+  }__e:${hasError}__c:${checked.toString()}`;
+  console.dir(storyId);
```
Comment (janiekyu) [inline packages/core/src/components/Form/Checkbox.stories.js]: NIT: did you mean to leave this in?

File: packages/core/src/components/Form/pin.module.css [hunk 4]
```diff
@@ -1,46 +1,216 @@
...
-.form-input {
-  composes: form-input from "common.module.css";
+.cds-form__label.cds-form--disabled {
+  color: var(--cds-form-disabled-label-color);
 }
 
-.form-field {
-  composes: form-field from "common.module.css";
+.cds-form {
+  /* HINT */
```
Comment (janiekyu) [inline packages/core/src/components/Form/pin.module.css:17]: Is this expected? I'm seeing this in multiple places.

**Review by janiekyu — APPROVED** [review]
A few non-blocking comments/questions. 

I think we should also update this component creating guide: https://www.notion.so/compassionux/4894f04a811c4e8eb45c1c20aacfaf18?v=151da545cc6c45fca4a239eb2a235caf&p=9c70fc25f7a44fa38b63e6bd3940c389&pm=s

Maybe we should create a separate Notion task for updating the doc?

File: lerna.json (hunk 2 above)
Comment (kendrick) [reply lerna.json:4]: You’re right that it’s automatically incremented using semver as part of Lerna’s publishing workflow, but that calls `npm version` (or something analogous) under the hood. That command both increments this value and creates a git tag at the new version number.

File: packages/core/scripts/build-sass.sh (hunk 1 above)
Comment (kendrick) [reply packages/core/scripts/build-sass.sh:7]: It’s the right call! 😄 Glad you mentioned it.

File: packages/core/src/components/Form/Checkbox.stories.js (hunk 3 above)
Comment (kendrick) [reply packages/core/src/components/Form/Checkbox.stories.js]: Eek! Great catch—thank you.

File: packages/core/src/components/Form/pin.module.css (hunk 4 above)
Comment (kendrick) [reply packages/core/src/components/Form/pin.module.css:17]: The `composes:` bits not being there are expected for the time being. This is a bit that we’ll/I’ll need to recreate in the “comprehensive” stylesheet someone would use if they were using `core` only. At the moment, the CSS Module `composes` output is recreated via the `form-common` import at the top of the [PIN SCSS](https://github.com/compassion-global-experience/compassion-design-system/blob/cf9549d0da582cab9fe0e6a3d2d386db0561e569/packages/core/src/components/Form/pin.scss).

File: packages/core/src/components/Form/Checkbox.stories.js (hunk 3 above)
Comment (kendrick) [reply packages/core/src/components/Form/Checkbox.stories.js]: (The ID value is intentional—but there may be a better way to convey the intent.)

**Pull request merged**